src/py/flwr/server/superlink/fleet/grpc_rere/server_interceptor.py

```diff
@@ -172,3 +171,55 @@ def _generic_method_handler(
             request_deserializer=method_handler.request_deserializer,
             response_serializer=method_handler.response_serializer,
         )
+
+    def _verify_node_id(self, node_id: int, request: Request) -> bool:
```

Comment:
If we make `node_id` an `Optional[int]` here, then the first check should be whether `node_id` is `None`

Reply:
Changed to optional.